### PR TITLE
feat: add all fetcher on FetcherName

### PR DESCRIPTION
**Pull request by matheushahnn**
## Context

use FetcherName for other fetchers - we already use it for uniswap, apply same stuff all other fetchers

## To-do list

- [x] Tested on dev
- [x] Changelog was updated with current changes

### CODE REVIEW

**Review by gagiuntoli — APPROVED** [review]
I am investigating the error in the CI

File: test/services/FeedProcessor.test.ts [hunk 1]
```diff
@@ -371,11 +371,15 @@ describe('FeedProcessor', () => {
           calculatorRepository.find.withArgs('Identity').returns(new IdentityCalculator());
 
           const feeds1: Feeds = {
-            TEST: feedFactory.build({inputs: [feedInputFactory.build({fetcher: {name: 'WrongFetcher'}})]}),
+            TEST: feedFactory.build({
+              inputs: [feedInputFactory.build({fetcher: {name: 'WrongFetcher' as FetcherName}})],
+            }),
```
Comment (gagiuntoli) [inline test/services/FeedProcessor.test.ts:376]: I assume that this test doesn't make sense anymore, before we were relying on strings and but with your improvements we accept only enums and there is no room for this kind of error. I would remove this kind of unit tests.

**Review by gagiuntoli — CHANGES_REQUESTED** [review]
Investigating CI failures

File: test/services/FeedProcessor.test.ts (hunk 1 above)
Comment (matheushahnn) [reply test/services/FeedProcessor.test.ts:376]: Yeah, I think we are safe form this error now. Even from feeds file we ensure with the feeds-schema that we are using only the enums

Comment (matheushahnn) [reply test/services/FeedProcessor.test.ts]: Done

**Review by gagiuntoli — APPROVED** [review]
Looks good now, my issues were addressed.

**Pull request merged**